app/routes/models/enrich-request.js

```diff
@@ -99,7 +99,7 @@ function ViewModel (payload, error) {
         }
```

Comment:
On line 84, `checked: getObjectKeyEquals(payload, 'debtType', IRREGULAR)` needs changed to `checked: getObjectKeyEquals(payload, 'debt-type', IRREGULAR)` and similar for line 89

Reply:
updated 'debtType' to 'debt-type'